folsom/src/main/java/com/spotify/folsom/MemcacheClient.java

```diff
@@ -180,4 +181,9 @@
    * @return the underlying raw memcache client.
    */
   RawMemcacheClient getRawMemcacheClient();
+
+  /** @return a snapshot of the currently connected nodes. */
```

Comment:
Perhaps explain what goes in the String keys.

Reply:
Done!